sdk/cosmos/azure-cosmos/src/main/java/module-info.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+module com.azure.cosmos {
+
+    requires transitive reactor.core;
+    requires transitive org.reactivestreams;
+
+    requires transitive com.fasterxml.jackson.annotation;
+    requires transitive com.fasterxml.jackson.core;
+    requires transitive com.fasterxml.jackson.databind;
```

Comment:
Don't need any of these as you are requiring com.azure.core which transitively requires these.

Reply:
Done.